kamon-stackdriver/src/main/scala/kamon/stackdriver/StackdriverSpanReporter.scala

```diff
@@ -65,66 +59,42 @@ class StackdriverSpanReporter extends SpanReporter {
       logger.error("Failed to close TraceServiceClient", error)
     }
 
-  private def writeTraces(traces: Traces): Unit = {
-    val request = PatchTracesRequest
-      .newBuilder()
-      .setProjectId(projectId)
-      .setTraces(traces)
-      .build()
+  private def convertSpan(span: FinishedSpan): Span = span match {
+    case FinishedSpan(context, operationName, start, end, tags, _) =>
+      val traceId = context.traceID.string
+      val spanId  = context.spanID.string
+      val name    = SpanName.of(projectId, traceId, spanId).toString
+      Span
+        .newBuilder()
+        .setName(name)
+        .setDisplayName(TruncatableString.newBuilder().setValue(operationName).build())
+        .setStartTime(instantToTimestamp(start))
+        .setEndTime(instantToTimestamp(end))
+        .setAttributes(Attributes.newBuilder().putAllAttributeMap(tagsToLabels(tags).asJava))
+        .setSpanId(spanId)
+        .setParentSpanId(context.parentID.string)
```

Comment:
I think this might be empty, eg. ` == IdentityProvider.NoIdentifier` will that be ok?

Reply:
`NoIdentifier.string` give us same value as default `parentSpanId` in `Span` so it should be ok